docs/dapps/integration/security-considerations.md

```diff
@@ -0,0 +1,120 @@
+Finally, you are welcome to review the contracts behind [Api3ReaderProxyV1](/dapps/integration/contract-integration.md#api3readerproxyv1), specifically [Api3ServerV1](https://github.com/api3dao/contracts/blob/main/contracts/api3-server-v1/Api3ServerV1.sol) and [Api3ServerV1OevExtension](https://github.com/api3dao/contracts/blob/main/contracts/api3-server-v1/Api3ServerV1OevExtension.sol).
+Note that our [contract developer docs](https://github.com/api3dao/contracts/tree/main/docs) may come in helpful in interpreting the design decisions made for these contracts.
+
+## Privileged accounts
+
+Api3ReaderProxyV1 is a [UUPS-upgradeable](https://eips.ethereum.org/EIPS/eip-1822) contract, which can be upgraded by [a 4-of-8 multisig](https://github.com/api3dao/contracts/blob/main/data/manager-multisig-metadata.json#L2) that is owned by members of the API3 technical team.
+This upgradeability feature is intended to be used only in exceptional occasions to:
+
+- Respond to newly discovered compiler, library or contract vulnerabilities,
+- Migrate users to potential new versions of the contracts.
+
+[A 4-of-4 multisig,](https://github.com/api3dao/contracts/blob/main/data/dapi-management-metadata.json#L2) which again is owned by members of the API3 technical team, approves the root of a Merkle tree that contains the data feed configurations.
+In other words, adding, removing or replacing API providers that contribute to the aggregation of data feeds requires signatures from all owners of this multisig.
+
+::: info ℹ️ Info
+
+The configuration of data feed sources depends on various factors such as uptime, accuracy, response time to incidents and qualitative considerations.
+The signers of the multisig have access to this data and are responsible to verify them.
+
+:::
+
+## Update parameters
+
+Once a plan has been purchased on API3 Market, the respective data feed will be updated with the advertised deviation threshold and heartbeat interval until the expiration of the plan.
+This depends on the API3 technical team to keep the wallets that will send the update transactions funded, and maintain the infrastructure that will use these wallets to send the update transactions.
+This operation is supported by dedicated monitoring personnel, automated alerting systems, and multiple redundant infrastructure layers.
+
+::: info ℹ️ Info
+
+We have been providing oracle services as early as [2019](https://etherscan.io/txs?a=0x78e76126719715eddf107cd70f3a31dddf31f85a&p=1029), and were listed as the [best responding oracle](https://raw.githubusercontent.com/api3dao/oev-docs/refs/heads/dapps-additions/docs/dapps/integration/images/reputation-link.png) among all Chainlink oracles by [`reputation.link`](https://www.google.com/search?q=%22reputation.link%22+chainlink) as of September 2020, which is when we published the [API3 whitepaper](https://github.com/api3dao/api3-whitepaper) and requested to be removed from Chainlink data feeds.
```

Comment:
TODO: Update the raw link to the image before merging. Also feel free to suggest a better way of doing this.